src/gosling-track/gosling-track.ts

```diff
@@ -413,29 +413,32 @@ function GoslingTrack(HGC: any, ...args: any[]): any {
                 // Apply filters
                 if (resolved.dataTransform !== undefined && IsDataTransform(resolved.dataTransform)) {
                     resolved.dataTransform.filter.forEach(filter => {
+                        const { field, not } = filter;
                         if (IsOneOfFilter(filter)) {
-                            const { field, oneOf, not } = filter;
+                            const { oneOf } = filter;
                             tile.tileData.tabularDataFiltered = tile.tileData.tabularDataFiltered.filter(
                                 (d: { [k: string]: number | string }) => {
-                                    return not
+                                    return not ?? false
```

Comment:
I think we can remove `?? false` to make the logical operation shorter/clearer.

```suggestion
                                    return not
```